packages/KLabel/KLabel.vue

```diff
@@ -12,6 +12,7 @@
         icon="help"
         height="16"
         width="16"
+        size="16"
```

Comment:
Can we remove the explicit definitions above for `height` and `width`?

Reply:
Removed ✔️